content/blog/open-sourcing-workshops-on-demand-part-5-create-a-workshop.md

```diff
@@ -0,0 +1,226 @@
+* **D﻿uration:** All workshops are time bombed. You will define here the time alloacted to perform the workshop.
+* **A﻿ctive:** Tag to set to enable visibility of the workshop's tile in the registration portal.
+
+I﻿f you feel you need more details about the registration process, please take a look at the **Register Phase** paragraph in [the following introductionary blog](https://developer.hpe.com/blog/willing-to-build-up-your-own-workshops-on-demand-infrastructure/).
+
+I﻿n the **Replays table:**
+
+A﻿ new entry will need the following:
+
+![](/img/wod-db-go-3.png)
+
+* **Id:** This entry will be filled with the name of a script to be compiled at deployment time. This feature allows for instance the admin to hide login scripts and credentials in non-editable executable files.
+* **Avatar:**  This defines whethere a workshop require some password variable to be leveraged or not.
+* **D﻿escription:** Same description as the one available in the workshop's table entry
+* **P﻿resenter:** Name of the presenter
+* **Role:** Roe of the presenter (the workshop developer in fact: Solution Architect, Fullstack Developer, etc...)
+* **V﻿ideoLink:** Youtube link of the recorded video to be used as a replay.
+* **W﻿orkshopId:** Id of the workshop linked to the video. 
+* **A﻿ctive:** Tag to set to enable visibility of the replay in registration portal.
+
+One could create these entries manually or leverage a simple wod.yml file containing them and that can be later parsed in order to feed the database with the relevant info, quite handy,no ?
+
+Here is an exampe of such a file:
+
+```
+%YAML 1.1
+# Meta data for the GO101 Workshop to populate seeder
+---
+name: 'GO 101 - A simple introduction to Go Programming Language'
+notebook: 'WKSHP-GO101'
+description: 'Go, also called Golang or Go language, is an open source programming language that Google developed. Software developers use Go in an array of operating systems and frameworks to develop web applications, cloud and networking services, and other types of software. This workshop will drive you through the basics of this programming language.'
+active: false
+capacity: 20
+priority: 1
+range: [151-170]
+reset: false
+ldap: false
+location: 'mougins'
+replayId: 31
+varpass: false
+compile: false
+workshopImg: 'https://us-central1-grommet-designer.cloudfunctions.net/images/frederic-passeron-hpe-com/WOD-GO-101-A-simp-introduction-to-Go-programming-language.jpeg'
+badgeImg: 'https://us-central1-grommet-designer.cloudfunctions.net/images/frederic-passeron-hpe-com/go101-a-simple-introduction-to-go-programming-language.jpg'
+beta: false
+category: 'Open Source'
+duration: 4
+alternateLocation: 'grenoble'
+presenter: 'Matthew Doddler'
+role: 'FullStack developer'
+avatar: '/img/SpeakerImages/MattD.jpg'
+videoLink: 'https://hpe-developer-portal.s3.amazonaws.com/Workshops-on-Demand-Coming-Soon-Replay.mp4'
+```
+
+A﻿s the developer of the Workshops-on-demand content, Matt had to perform several tasks:
```

Comment:
"Workshops-on-Demand"

The reader is going to say "Who's Matt? Where did he come in?" This is the first place I've seen his name (unless I missed it earlier?). You'll want to introduce his name in the context of who he is.